sdks/python/build.gradle

```diff
@@ -222,11 +222,16 @@ def portableWordCountTask(name, streaming) {
               "--runner=PortableRunner",
       ]
 
+      // Workaround for ensuring writes to the local file system
+      // stay within the same Docker container. Otherwise the Write
+      // logic would throw because files can't be shared between
+      // different executable stages.
+      options += ["--environment_cache_millis=60000"]
+
       if (streaming) {
         options += ["--streaming"]
+        options += ["--parallelism=8"]
```

Comment:
Shall we keep it to a lower number to avoid scheduling issue between sdkharness and flink?

Reply:
I thought it is only an issue in batch mode. Have we seen this in streaming?

Reply:
If it is the issue is parallelism then just make it 1. But it isn't clear yet which setting causes the issue.

Reply:
No, this is just to fix the parallelism for repeatable tests. The parallelism is otherwise 16 on most Jenkins hosts.